torchdata/datapipes/iter/load/online.py

```diff
@@ -110,6 +110,8 @@ def _get_response_from_google_drive(url: str, *, timeout: Optional[float]) -> Tu
         if confirm_token is None:
             if "Quota exceeded" in str(response.content):
                 raise RuntimeError(f"Google drive link {url} is currently unavailable, because the quota was exceeded.")
+            if "Download anyway" in str(response.content):
```

Comment:
Could you please take a reference from TorchVision to make this function more robust?
See:
- https://github.com/pytorch/vision/blob/ac56f52e6d6788c31726ee2d3a55afa442c6c3ec/torchvision/datasets/utils.py#L201-L214
- https://github.com/pytorch/vision/blob/ac56f52e6d6788c31726ee2d3a55afa442c6c3ec/torchvision/datasets/utils.py#L244-L261

Reply:
I've revised the patch to more closely match the torchvision method. I've made sure it works for the "Virus scan warning" case but I'm not sure how to test the quota exceeded case. In any case, I believe the flow in my updated change should match what's in the torchvision files you sent.